tests/integration/Microsoft.Bot.Builder.Integration.AspNet.Core.Tests/CloudAdapterTests.cs

```diff
@@ -194,7 +194,7 @@ public async Task InjectCloudEnvironment()
             {
                 ClaimsIdentity = new ClaimsIdentity(),
                 ConnectorFactory = new TestConnectorFactory(),
-                Scope = "scope",
+                Audience = "scope",
```

Comment:
Minor nit, can you clean this up by changing `"scope"` to `"outboundAudience"`?

Reply:
I'm going to look through all the tests - actually some of these might even be failing

Reply:
i forgot this one - next time

Reply:
done!